lib/vdsm/storage/lvmconf.py

```diff
@@ -184,3 +189,27 @@ def save(self):
     def close(self):
         log.debug("Closing LVM configuration %s", self.path)
         self.aug.close()
+
+
+def configured_value(option):
```

Comment:
I guess the user needs to specify now the string "section/option"?

Should we use section, option arguments?

One advantage is that we don't have to handle the case of reading the section. We can do:

    def configured_value(section, option):
        if not section:
            raise ValueError("Empty section")
        if not option:
            raise ValueError("Empty option")

And developer errors will be reported very clearly.

Another option is to use a string, and validate that the string always has
one "/", but this may assume too much about options name, maybe lvm will support
options with "/".